docker/opensource.md

```diff
@@ -86,6 +86,18 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # composer install
 # phpunit
 ```
-
+## Renormalizer
+https://github.com/shuaigroup/Renormalizer
+
+$ git clone https://github.com/shuaigroup/Renormalizer.git
+$ cd Renormalizer
+$ docker run -it --name Renormalizer-dev -w /opt -v %cd%:/opt python:latest bash
+$ pip install renormalizer
+$ pip install --upgrade pip
+$ pip install qutip
+$ pip install recommonmark
+$ pip install Yaml8
+$ pip install -r requirements.txt
```

Comment:
I get

ERROR: Ignored the following versions that require a different python version:....